examples/zero-click-loki/2-loki-tls.yaml

```diff
@@ -44,6 +44,8 @@ data:
             max_size_bytes: 500MB
             validity: 24h
       parallelise_shardable_queries: true
+    query_scheduler:
+      max_outstanding_requests_per_tenant: 10000
```

Comment:
Do we really need 10k here ?

I tested 2048 previously and it was fine. LLC cluster is currently using this value.

Reply:
Let's try having all the team playing with LLC at the same time, and see how it goes :)